### PR TITLE
Accept the `$JUNIT_REPORT` flag in networking tests

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

[test]

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Hmm... we don't seem to grab the `/tmp/openshift/test-extended` directory. I've added some debugging .. let's see more tests

File: test/extended/networking.sh [hunk 1]
```diff
@@ -318,6 +318,13 @@ else
 
   os::util::environment::setup_tmpdir_vars "test-extended/networking"
 
+  # Allow setting $JUNIT_REPORT to toggle output behavior
+  if [[ -n "${JUNIT_REPORT:-}" ]]; then
+    export JUNIT_REPORT_OUTPUT="${LOG_DIR}/raw_test_output.log"
```
Comment (marun) [inline test/extended/networking.sh]: I think the raw output should vary by plugin, as the regular output already does or the output of only one of the plugins will be saved:

https://github.com/openshift/origin/blob/master/test/extended/networking.sh#L167

Comment (stevekuznetsov) [inline test/extended/networking.sh]: This is for the logging from the `os::cmd` tests, if there are any, not for the jUnit from Ginkgo.

**Comment by stevekuznetsov** [comment]
@marun can you affirm that the jUnit results we got from https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_extended_networking_minimal/103/testReport/ are correct?

**Comment by stevekuznetsov** [comment]
[merge] we'll at least get some, if not all of them this way

**Pull request merged**